dashboard/apps/macro_manager.fma/index.html

```diff
@@ -26,7 +26,7 @@ <h1><a id="app-header" href="">Macros</a></h1>
                 <table class="small-12 columns" style="width:100%" id="macro_table">
                 <tr>
                         <th></th>
-                        <th>Macro #</th>
+                        <th>Custom #</th>
```

Comment:
Why does this one just say "Custom" when all the others say "Custom Macro" or some equivalent?

Reply:
Did you change it? Or have you stopped using the branch and I can change it on your branch?